### ISSUE TITLE
Skipping player response from ios client (got player response for video "M5t4UHllkUM" instead of "2pFwQiwRbcg")

### ISSUE THREAD

**Issue by schmorp** [yt-dlp/yt-dlp#8812]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

yt-dlp asked me to report this issue. The problem affects a lot of videos, not just this one, so it could be an issue on youtubes side. Nevertheless, the video does play in chromium on the same box, and the download still fails when I try various proxies.

What makes it especially bad is that, when used with --download-archive, this results in the video marked as being downloaded when it isn't.

Interestingly, the "other" video (youtu.be/M5t4UHllkUM) is a weird video saying "VBideo not available", so this might be soem weird blocking mechanism (I downloaded a lot of videos a few days ago). In that case, it would still be nice if yt-dlp would simply fail instead of marling the video as successfully downloaded, possibly by hardcoding the M5t4UHllkUM video id.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.youtube.com/watch?v=2pFwQiwRbcg']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.12.19.232701 from yt-dlp/yt-dlp-nightly-builds [db8b4edc7] (zip)
[debug] Python 3.11.7 (CPython x86_64 64bit) - Linux-6.1.0-15-amd64-x86_64-with-glibc2.36 (OpenSSL 3.0.11 19 Sep 2023, glibc 2.36)
[debug] exe versions: ffmpeg 6.0.1 (fdk,setts), ffprobe 6.0.1, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, requests-2.28.1, sqlite3-3.40.1, urllib3-1.26.12, websockets-10.4, xattr-0.10.1
[debug] Proxy map: {'no': '127.0.0.1,localhost,cerebro,fuji,doom,127.0.0.0/8,10.0.0.0/8,192.168.0.0/16', 'xv_open': '/root/s/xv_open_proxy'}
[debug] Request Handlers: urllib
[debug] Loaded 1796 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Latest version: nightly@2023.12.19.232701 from yt-dlp/yt-dlp-nightly-builds
yt-dlp is up to date (nightly@2023.12.19.232701 from yt-dlp/yt-dlp-nightly-builds)
[youtube] Extracting URL: https://www.youtube.com/watch?v=2pFwQiwRbcg
[youtube] 2pFwQiwRbcg: Downloading webpage
[youtube] 2pFwQiwRbcg: Downloading ios player API JSON
WARNING: [youtube] Skipping player response from ios client (got player response for video "M5t4UHllkUM" instead of "2pFwQiwRbcg"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube] 2pFwQiwRbcg: Downloading android player API JSON
WARNING: [youtube] Skipping player response from android client (got player response for video "M5t4UHllkUM" instead of "2pFwQiwRbcg"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube] Skipping player response from web client (got player response for video "M5t4UHllkUM" instead of "2pFwQiwRbcg"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: Only images are available for download. use --list-formats to see them
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 2pFwQiwRbcg: Downloading 1 format(s): sb0
[debug] Invoking mhtml downloader on "https://i.ytimg.com/sb/M5t4UHllkUM/storyboard3_L2/M$M.jpg?sqp=-oaymwENSDfyq4qpAwVwAcABBqLzl_8DBgi867OjBg==&sigh=rs$AOn4CLD6zz6wUMPtwBuO9zjkyt-TR6iF_g"
[download] Video Not Available [2pFwQiwRbcg].mhtml has already been downloaded
[download] 100% of   20.08KiB
```

**Comment by bashonly** [comment]
Duplicate of #7143

Your IP address has been blocked by Youtube

**Comment by bashonly** [comment]
1. trying forcing ipv4 with `-4` or ipv6 with `-6`
2. change your IP
3. wait longer
